lib/src/Base/Stat/HMatrixImplementation.cxx

```diff
@@ -307,6 +307,34 @@ void HMatrixImplementation::assemble(const HMatrixTensorRealAssemblyFunction& f,
 #endif
 }
 
+/* Compute an approximation of the largest eigenValue (in magnitude) using power iterations */
+Scalar HMatrixImplementation::computeApproximateLargestEigenValue(const Scalar epsilon)
+{
+  const UnsignedInteger dimension = getNbRows();
+  Point currentEigenVector(dimension, 1.0);
+  Point nextEigenVector(dimension);
+  gemv('N', 1.0, currentEigenVector, 0.0, nextEigenVector);
+  Scalar nextEigenValue = nextEigenVector.norm();
+  Scalar currentEigenValue = nextEigenValue / std::sqrt(dimension);
```

Comment:
this line fails on appveyor, it should be:
std::sqrt(1.0 * dimension)

Reply:
Ok done.